### PR TITLE
[DOCS] Update misc.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Review by fulldecent — APPROVED** [review]
I performed a code-level review

File: docs/miscellaneous.rst [hunk 1]
```diff
@@ -322,43 +321,39 @@ The following is the order of precedence for operators, listed in order of evalu
 Global Variables
 ================
```
Comment (leonardoalt) [inline docs/miscellaneous.rst:322]: This section has its own bigger section `units-and-global-variables`. Do we want to keep both?

File: docs/miscellaneous.rst [hunk 2]
```diff
@@ -322,43 +321,39 @@ The following is the order of precedence for operators, listed in order of evalu
...
+- ``abi.encode(...) returns (bytes memory)``: :ref:`ABI <ABI>`-encodes the given arguments
+- ``abi.encodePacked(...) returns (bytes memory)``: Performs :ref:`packed encoding <abi_packed_mode>` of the given arguments
+- ``abi.encodeWithSelector(bytes4 selector, ...) returns (bytes memory)``: :ref:`ABI <ABI>`-encodes the given arguments
    starting from the second and prepends the given four-byte selector
-- ``abi.encodeWithSignature(string signature, ...) returns (bytes)``: Equivalent to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature)), ...)```
-- ``block.blockhash(uint blockNumber) returns (bytes32)``: hash of the given block - only works for 256 most recent, excluding current, blocks - deprecated in version 0.4.22 and replaced by ``blockhash(uint blockNumber)``.
+- ``abi.encodeWithSignature(string memory signature, ...) returns (bytes memory)``: Equivalent to ``abi.encodeWithSelector(bytes4(keccak256(bytes(signature)), ...)```
 - ``block.coinbase`` (``address payable``): current block miner's address
```
Comment (leonardoalt) [inline docs/miscellaneous.rst:330]: `blockhash()` missing

File: docs/miscellaneous.rst (hunk 1 above)
Comment (erak) [reply docs/miscellaneous.rst:322]: I think it's good to document them in their own section and also here as part of the cheat sheet.

File: docs/miscellaneous.rst (hunk 2 above)
Comment (erak) [reply docs/miscellaneous.rst:330]: This change is correct. The alias ``block.blockhash()`` was removed and ``blockhash()`` is mentioned below 😉

**Review by erak — APPROVED** [review]

File: docs/miscellaneous.rst (hunk 2 above)
Comment (leonardoalt) [reply docs/miscellaneous.rst:330]: Ah true, didn't see `blockhash()` down the list.

**Pull request merged**